src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -0,0 +1,25 @@
+import lotto.domain.Lotto
+import lotto.domain.LottoNumber
+import lotto.domain.Rank
+
+class Lottos(private val tickets: List<Lotto>) {
+    init {
+        require(tickets.isNotEmpty()) { "로또는 최소 1개 이상 구매해야 합니다." }
+    }
+
+    val size: Int
+        get() = tickets.size
+
+    val lottos: List<Lotto>
+        get() = tickets.toList()
+
+    fun matchNumber(
+        winningNumber: Lotto,
+        bonusBall: LottoNumber,
+    ): Map<Rank, Int> {
+        return tickets
+            .map { it.match(winningNumber, bonusBall) }
+            .groupBy { it }
+            .mapValues { it.value.size }
```

Comment:
`grounpingBy`를 활용하도록 변경해볼 수 있을 것 같네요. 😃
```suggestion
        return tickets
            .map { it.match(winningNumber, bonusBall) }
            .groupingBy { it }
            .eachCount()
```

Reply:
groupping by를 사용하니 더 깔끔하고 가독성이 좋아진것 같습니다! ㅎㅎ 감사합니다.